sources/org.osbuild.ostree

```diff
@@ -89,13 +103,18 @@ class OSTreeSource(sources.SourceService):
         gpg = remote.get("gpgkeys", [])
         uid = str(uuid.uuid4())
 
-        verify_args = []
+        remote_add_args = []
         if not gpg:
-            verify_args = ["--no-gpg-verify"]
+            remote_add_args = ["--no-gpg-verify"]
+
+        if remote.get("secrets", {}).get("name") == "org.osbuild.rhsm-consumer":
```

Comment:
I think I would have called it `org.osbuild.rhsm.consumer`.

Reply:
done!